hermes/bin/src/runtime_extensions/hermes/http_gateway/routing.rs

```diff
@@ -145,7 +145,7 @@ async fn route_to_hermes(
     }
 
     let (_parts, body) = req.into_parts();
-    if uri.path() == WEBASM_ROUTE {
+    if uri.path() == WEBASM_ROUTE || uri.path().starts_with(&format!("{WEBASM_ROUTE}/")) {
```

Comment:
I think we also need to take care of the case sensitivity here. The [docs](https://docs.rs/http/1.3.1/http/uri/struct.Uri.html#method.path) for `path()` say that _"The path component is case sensitive."_